src/Engine/Component/CameraComponents/CameraController.cpp

```diff
@@ -4,33 +4,52 @@
 #include "IO/Keyboard.hpp"
 #include "Component/SpatialComponents/SpatialComponent.hpp"
 
+// Debug
+#include <stdio.h>
+
 void CameraController::init() {
 
 }
 
 void CameraController::update(float dt) {
+    /* Checks if the camera direction or location has been modified */
+    bool modified;
```

Comment:
This boolean doesn't seem to do anything. Also, this logic should go in the Camera, not the CameraController. Also, I already added this logic to Camera.

Reply:
Ok. This can be removed